server/channels/app/helper_test.go

```diff
@@ -249,36 +245,22 @@ func SetupWithClusterMock(tb testing.TB, cluster einterfaces.ClusterInterface) *
 	return setupTestHelper(dbStore, true, true, nil, []Option{SetCluster(cluster)}, tb)
 }
 
-var initBasicOnce sync.Once
-var userCache struct {
-	SystemAdminUser *model.User
-	BasicUser       *model.User
-	BasicUser2      *model.User
-}
-
 func (th *TestHelper) InitBasic() *TestHelper {
-	// create users once and cache them because password hashing is slow
-	initBasicOnce.Do(func() {
-		th.SystemAdminUser = th.CreateUser()
-		th.App.UpdateUserRoles(th.Context, th.SystemAdminUser.Id, model.SystemUserRoleId+" "+model.SystemAdminRoleId, false)
-		th.SystemAdminUser, _ = th.App.GetUser(th.SystemAdminUser.Id)
-		userCache.SystemAdminUser = th.SystemAdminUser.DeepCopy()
-
-		th.BasicUser = th.CreateUser()
-		th.BasicUser, _ = th.App.GetUser(th.BasicUser.Id)
-		userCache.BasicUser = th.BasicUser.DeepCopy()
-
-		th.BasicUser2 = th.CreateUser()
-		th.BasicUser2, _ = th.App.GetUser(th.BasicUser2.Id)
-		userCache.BasicUser2 = th.BasicUser2.DeepCopy()
-	})
-	// restore cached users
-	th.SystemAdminUser = userCache.SystemAdminUser.DeepCopy()
-	th.BasicUser = userCache.BasicUser.DeepCopy()
-	th.BasicUser2 = userCache.BasicUser2.DeepCopy()
+	// create users
+	th.SystemAdminUser = th.CreateUser()
+	_, appErr := th.App.UpdateUserRoles(th.Context, th.SystemAdminUser.Id, model.SystemUserRoleId+" "+model.SystemAdminRoleId, false)
+	require.Nil(th.TB, appErr)
+	th.SystemAdminUser, _ = th.App.GetUser(th.SystemAdminUser.Id)
+
+	th.BasicUser = th.CreateUser()
+	th.BasicUser, _ = th.App.GetUser(th.BasicUser.Id)
+
+	th.BasicUser2 = th.CreateUser()
+	th.BasicUser2, _ = th.App.GetUser(th.BasicUser2.Id)
```

Comment:
The tests are failing with 

```
    helper_test.go:263: 
        	Error Trace:	/mattermost/server/channels/app/helper_test.go:263
        	            				/mattermost/server/channels/app/webhook_test.go:26
        	Error:      	Received unexpected error:
        	            	pq: duplicate key value violates unique constraint "users_pkey"
        	Test:       	TestCreateIncomingWebhookForChannel
```


an you please add some error checks, please and look into the test failure? Thanks!